### ISSUE TITLE
How can I set the datasets for super resolution

### ISSUE THREAD

**Issue by appleleaves** [ofsoundof/dhp#2]
Thanks for sharing!
There are some important issues I want to ask and I look forward to your reply.

1. DIV2KSUB is a subset that not mentioned.

2. The Gray version of DIV2K is not mentioned.

3. When using DIV2K for the training set, the code shows that the validation set is used, but the training set is not used, am I missing something?

4. Many npy files are not available but seem to be used in the codes. 

Thanks a lot!
